assets/sass/components/settings/_googlesitekit-settings-notice.scss

```diff
@@ -204,3 +212,31 @@
 .googlesitekit-settings-notice-ua-cutoff-warning__notice {
 	margin: 0 14px 0 0;
 }
+
+.googlesitekit-settings-notice-adsense-top-earning-pages-widget {
+	align-items: center;
+	padding-bottom: 24px;
+	padding-top: 24px;
+
+	.googlesitekit-settings-notice__icon {
+		margin-right: 20px;
+
+		svg {
+			background-color: $c-surfaces-on-surface;
+			border-radius: 50%;
+			padding: 2px;
```

Comment:
This might not be needed once the svg is updated

Reply:
Removed as we're using a new SVG.

Reply:
Removed as we're using a new SVG.